test/e2e/scenario/rum/actions.scenario.ts

```diff
@@ -161,6 +161,7 @@ describe('action collection', () => {
 
       expect(resourceEvents.length).toBe(1)
       expect(resourceEvents[0].action!.id).toBe(actionEvents[0].action.id!)
+      expect(resourceEvents[0].resource.first_byte).toBeDefined()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what is the reasoning for adding this check in this specific test?